db/migrations/20220109181508_session.sql

```diff
@@ -0,0 +1,20 @@
+-- +goose Up
+CREATE TABLE IF NOT EXISTS public.session
+(
+    "session_ID" integer NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
```

Comment:
`GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 )`

What is the reason you are using this to set up the id column?

Reply:
why not? it is pgadmin generated code